controllers/external-accounts.js

```diff
@@ -69,6 +72,19 @@ const linkExternalAccount = async (req, res) => {
       userId
     );
 
+    const unverifiedRoleRemovalResponse = await removeDiscordRoleUtils.removeDiscordRole(
+      req.userData,
+      attributes.discordId,
+      undefined,
+      DISCORD_ROLES.UNVERIFIED
+    );
+
+    if (!unverifiedRoleRemovalResponse.success) {
+      return res.status(500).json({
+        message: `User details updated successfully but ${unverifiedRoleRemovalResponse.message}. Please contact admin`,
```

Comment:
- if the action failed how did is "user details updated successfully" the correct sentence?

Reply:
Removed the word "successfully". Hope it is correct now. Please let me know if the message makes sense now.